src/components/editor/Bindings/Selector.tsx

```diff
@@ -176,13 +176,15 @@ function BindingSelector({
                 if (currentConfig.errors.length > 0) {
                     return (
                         <>
-                            <WarningCircle
-                                style={{
-                                    marginRight: 4,
-                                    fontSize: 12,
-                                    color: theme.palette.error.main,
-                                }}
-                            />
+                            <Box>
+                                <WarningCircle
```

Comment:
Curious if a defined min width on the icon is an alternative to prevent shrinkage here.

Reply:
Potentially... however I think we'll need to decide if we want to use `style` for icons or include a wrapper that allows us to style with `sx`